frontend/public/components/utils/resource-link.tsx

```diff
@@ -80,14 +80,16 @@ export const ResourceLink = connectToModel(
     hideIcon,
     title,
     children,
+    noPadding = false,
```

Comment:
Don't do it this way. Pass in a `className` and write some CSS that targets the dialog specifically.